src/components/Cards/Cards.jsx

```diff
@@ -1,28 +1,21 @@
 import Card from "../Card/Card";
+import automobile from "../../data/data.json";
 
 import "./Cards.scss";
 
 export default function Cards() {
   return (
     <div className="cards__panel">
-      <Card
-        imageType="sedan"
-        title="SEDANS"
-        text="Choose a sedan for its affodability and excellent fuel economy. Ideal for cruising in the city or your next road trip."
-        buttonColor="orange"
-      />
-      <Card
-        imageType="suvs"
-        title="SUVS"
-        text="Take an SUV for its spacious interior, power, and versatility. Perfect for your next family vacation and off-road adventures."
-        buttonColor="green"
-      />
-      <Card
-        imageType="luxury"
-        title="LUXURY"
-        text="Cruise in the best car brands without the bloated prices. Enjoy the enhanced comfort of a luxury rental and arrive in style"
-        buttonColor="blue"
-      />
+      {automobile.map((props) => {

```

Comment:
now you can change from `automobile` to `data`. 
The object that you received on `data` is something like:

```js
{
  automobile: [...] // All three inside this array
}
```

So what is missing is to access the key that is responsible to keep the array.

so in object notation you can use `.` or `[]` to access:

```js
const data = {
  content: [1,2,3]
};

console.log(data.content) // [1,2,3]
// or
console.log(data["content"]) // [1,2,3]
```

and data.content has .map to iterate over the data.

```js
data.content.map()
```